recipe/meta.yaml

```
@@ -11,8 +11,9 @@ source:
 
 build:
   number: 1
-  script: NPY_NUM_BUILD_JOBS=1 {{ PYTHON }} -m pip install . -vvv  # [linux and ppc64le]
-  script: {{ PYTHON }} -m pip install . -vvv  # [not (linux and ppc64le)]
+  script:
+    - export NPY_NUM_BUILD_JOBS=${CPU_COUNT}
+    - {{ PYTHON }} -m pip install . -vvv
```

Don't you need this instead?
```suggestion
  script:
    - export CMAKE_BUILD_PARALLEL_LEVEL=${CPU_COUNT}
    - {{ PYTHON }} -m pip install . -vvv
```

---

No, this is a true Setuptools build, not a CMake-Setuptools one. We technically can call this variable anything we want to, though - and maybe "CMAKE_BUILD_PARALLEL_LEVEL" is better than "NPY_NUM_BUILD_JOBS"; the latter was taken because we used to use NumPy's parallel build tooling, and we kept the name when we dropped the NumPy build. But maybe "CMAKE_BUILD_PARALLEL_LEVEL" is more of a standard.